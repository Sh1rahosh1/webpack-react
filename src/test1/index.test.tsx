import { shallow, mount } from 'enzyme';
import Test1 from './index'
import React from 'react';


describe('load page', () => {
    let subject;
    it('should map snapshot', () => {
        subject = shallow(<Test1 />)
        expect(subject).toMatchSnapshot()
    })

    it('should change state when click button', () => {
        subject = mount(<Test1 />);
        subject.find('Button').at(0).simulate('click');
        subject.find('Button').at(1).simulate('click');
        expect(subject.find('.testDiv').at(0).text()).toEqual('3')
        subject.debug();
        expect(subject.state().count).toEqual(3)
    })
})
