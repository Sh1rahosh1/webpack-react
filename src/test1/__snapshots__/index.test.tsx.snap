// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`load page should map snapshot 1`] = `
<div>
  <Button
    block={false}
    ghost={false}
    htmlType="button"
    loading={false}
    onClick={[Function]}
  >
    1
  </Button>
  <Button
    block={false}
    ghost={false}
    htmlType="button"
    loading={false}
    onClick={[Function]}
  >
    2
  </Button>
  <div
    className="testDiv"
  >
    1
  </div>
</div>
`;
